### PR TITLE
Update dependency and improve user experience

### PR DESCRIPTION
**Pull request by amjad-adsk**
The changes in this PR are intended to fix some of the known problems we have:

- [x] Outdated SDK dependency
- [x]  Remove Skypack as a package loader
- [x] Improve UI to give users a better understanding of what's going on
- [x] Let the user know when the file has been added to the library
- [x] Remove the `local_testing` extension reference
- [x] Remove the `left_panel` reference
- [x] Improve the overall `readme` file

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -6,10 +6,14 @@
     <script type="importmap">
       {
         "imports": {
-          "forma-embedded-view-sdk/auto": "https://cdn.skypack.dev/forma-embedded-view-sdk@^0.3/auto"
+          "forma-embedded-view-sdk/auto": "https://esm.sh/forma-embedded-view-sdk@0.x/auto"
         }
       }
     </script>
```
Comment (magnus-poppe-adsk) [inline src/index.html]: I would prefer moving this import into the `src/sdk.js` as that would be consistent with how we recommend using this package: https://app.autodeskforma.com/forma-embedded-view-sdk/docs/index.html

**Review by magnus-poppe-adsk — APPROVED** [review]

File: src/index.html (hunk 1 above)
Comment (henrik-steen-adsk) [reply src/index.html]: It's better to recommend using import maps as we avoid the situation that you use different versions of the package. Keep in mind that the example provided in the SDK is the least required to do anything - although we might recommend a importmap there

File: src/index.html [hunk 2]
```diff
@@ -1,15 +1,11 @@
 <html>
   <head>
     <title>kenney.nl file browser</title>
```
Comment (henrik-steen-adsk) [inline src/index.html:3]: Based on changes in README this should probably be changed too

Comment (amjad-adsk) [reply src/index.html:3]: @henrik-steen-adsk, are you pointing towards the `title`?

File: src/index.html (hunk 1 above)
Comment (amjad-adsk) [reply src/index.html]: If we are planning to introduce the `importmap`, then we should do that in both repositories! Is that mandatory, I mean, there could exist two different valid ways to achieve the same objective, Right?

File: src/index.html (hunk 2 above)
Comment (henrik-steen-adsk) [reply src/index.html:3]: Yes

**Pull request merged**